src/types/types.js

```diff
@@ -456,8 +456,11 @@ export type EdgeDataDump = {
 
 export type EdgeFreshAddress = {
   publicAddress: string,
+  nativeAmount: string,
   segwitAddress?: string,
-  legacyAddress?: string
+  legacyAddress?: string,
+  segwitNativeAmount?: string,
+  legacyNativeAmount?: string
```

Comment:
We have some confusion here, since we were using `nativeAmount` before to indicate the amount requested, but now we are using it to indicate the balance. So maybe we should call these fields `nativeBalance?: string`, making them optional so we can tell the difference between 0 and "I don't know".

Otherwise, if you know for sure that we will never care about the difference between 0 and "not reported", we could make the balances mandatory, but then we would need fallback code to upgrade the responses from plugins that haven't been upgraded yet.

Reply:
Ah. I never realized that it meant amount requested! Sam pointed it out and he thought it was the address balance. I'll change to `nativeBalance` and make them all optional.